<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManiForm</class>
 <widget class="QWidget" name="ManiForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>596</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>二维码生成与识别</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1071</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 150, 200, 65), stop:0 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>生成二维码</string>
    </attribute>
    <widget class="QPushButton" name="saveQR">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>保存图片</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textForQR">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>请输入字符，以便生成二维码……</string>
     </property>
    </widget>
    <widget class="QPushButton" name="makeQR">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>生成二维码</string>
     </property>
    </widget>
    <widget class="QrcodeGenerate" name="showQR" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>511</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>识别二维码</string>
    </attribute>
    <widget class="QLabel" name="showPic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>500</width>
       <height>500</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>这里显示您打开的带有二维码的图片</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="openPic">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="showText">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>370</y>
       <width>521</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>这里会展示二维码识别的结果:)</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="goodQR">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>10</y>
       <width>350</width>
       <height>350</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>图片里截取的二维码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QrcodeGenerate</class>
   <extends>QWidget</extends>
   <header location="global">qrcodegenerate.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
